package.json

```diff
@@ -40,6 +40,7 @@
     "bluebird": "3.0.5",
     "body-parser": "1.14.2",
     "browserify": "14.0.0",
+    "browserify-istanbul": "^2.0.0",
```

Comment:
It's actually not a regex :) See https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json
Also, this line was added automatically by running "npm install browserify-istanbul --save-dev"